src/status_im/utils/ethereum/tokens.cljs

```diff
@@ -382,7 +382,7 @@
       {:name     "Aragon Test Token"
        :symbol   :ATT
        :decimals 8
-       :address  "0xfff28fb56b2a53597c55e1a767c906f339335025"}
+       :address  "0x00a8e52df8f4f1f4b67bded9ae6090b35489a973"}
       {:name     "Handy Test Token"
        :symbol   :HND
```

Comment:
:ATT ?

Reply:
what's wrong with that?

Reply:
it should be `ATT` instead of `HND`,  no?

Reply:
@asemiankevich Correct values are above, not bellow ;)

Reply:
@jeluard right! 👀